cylc/flow/task_job_mgr.py

```diff
@@ -997,14 +998,18 @@ def _set_retry_timers(
     def _simulation_submit_task_jobs(self, itasks, workflow):
         """Simulation mode task jobs submission."""
         for itask in itasks:
+            if not itask.mode_settings:
+                itask.mode_settings = ModeSettings()
+            itask.mode_settings.update(
+                itask, self.task_events_mgr.broadcast_mgr)
```

Comment:
This can either be set at simulation time or at sim_time_check time?

Reply:
I don't think they should be no, but I see that's what I've done. I'll assume that I'm wrong and remove them.